src/browser/tab/mod.rs

```diff
@@ -380,4 +377,34 @@ impl<'a> Tab {
         })?;
         Ok(self)
     }
+
+    /// Tells Chrome to start tracking which lines of JS have been executed.
+    ///
+    /// Equivalent to hitting the record button in the "coverage" tab in Chrome DevTools.
+    /// See the file `tests/coverage.rs` for an example.
+    pub fn start_js_coverage(&self) -> Result<&Self, Error> {
+        self.call_method(profiler::methods::Enable {})?;
```

Comment:
We may want to unhinge the `Enable`-call into a separate public method. It will always result in a clear error message if `Profiler.enable` has not been called, which makes forgetting to call this easy to debug. Unhinging makes it possible to implement `Profile.disable`, which people may want to use e.g. for performance reasons.

Reply:
Yep, will do.